### ISSUE TITLE
Domain activation is Failed urgent - CNAME already in use

### ISSUE THREAD

**Issue by levin-roski** [aws-amplify/amplify-console#167]
**App Id**: d3k3gb26krpwdc
**Region**: eu-central-1
**Step**: Domain activation
**Status**: Failed

>**Note**: Do not include information that is sensitive in nature such as your domain name, company etc.
![2019-09-30_150034](https://user-images.githubusercontent.com/18506470/65881232-67a3da80-e393-11e9-9817-be8bd5fe7a78.png)
![2019-09-30_150240](https://user-images.githubusercontent.com/18506470/65881240-696d9e00-e393-11e9-84d2-09df96981a47.png)

**Issue/question**
I cant add this domain, its managed by Route53 and everyting got setup automatically. I checked the docs and the other similiar issues here in GitHub but cant find any solution. I tried to change the CNAME to my amplify Subdomain instead of the automatically added cloudfront, which didnt work too.

Please help as soon as possible, as the site should be online ASAP.

**Comment by swaminator** [comment]
@levin-roski can you check your CloudFront console as mentioned here: https://docs.aws.amazon.com/amplify/latest/userguide/custom-domains.html#cnamealreadyexistsexception-error

**Comment by levin-roski** [comment]
Hi @swaminator thanks for your fast reply,

I checked the troubleshooting:
1. Yes the domain and subdomain is not in use by any other Clodufront distro, in fact I just added it for Amplify to Route53, its not used in any other AWS service.
2. Its not used by any Amplify app.
3. I tried multiple times, but it was never successfully setup, before I deleted the domain in Amplify. Maybe this is the issue currently and it will work tomorrow morning, cause caches get cleared.

I saw that there are many new issues with this similar problem today.
Somehow I guess the entries automatically added to Route53 by Amplify are wrong. They all point to a CloudFront distro, which I cant find in my console (probably the CloudFront behind the Amplify app), but if check the docs for external DNS-management it says to add a CNAME pointing to the Amplify subdomain, not the CloudFront distro directly like it was auto-setup in my Route53.

I hope it will be fixed tomorrow morning, as the site is not available anymore.

**Comment by levin-roski** [comment]
Hey @swaminator,

the issue still persists. The subdomain and domain is not used by anything else but this one Amplify app. Screenshots of the Route53 zone and the Amplify settings are attached in the first issue comment.
How can I get support? And how long does it take to be become available?

As its a production site, one solution will be to host it myself with S3 static site hosting and Cloudfront in front, as the custom domain is a crusial feature. Amplify is really nice and worked good till this point, but cant be used by me at the moment, as the website needs a custom domain.

Thanks!
Levin

**Comment by garyleefight** [comment]
@levin-roski Hi, your subDomain started with www is attached to cloudfront distribution:
d3rinca8bwi4kg.cloudfront.net
in your account. Please delete it first. The account ID is: 6354xxxx7815

**Comment by levin-roski** [comment]
@garyleefight that entry was created by Amplify automatically, even if I replace it with the apps Amplify domain, it doesn't activate the domain and A-Record mapping is just possible to the cloudfront related to the Aplify app.

@swaminator I deleted the app, deleted all Route53 records for the domain-zone and tried to create the app in Amplify from scratch. Still didn't work.
I will host it with other AWS services now. If you want me to try again, or if you think Amplify custom domains work again, please inform me, then I can try to create the Amplify from scratch again and route the custom domain.

**Comment by levin-roski** [comment]
@swaminator @garyleefight figured out the issue! It's was not an issue of Amplify at all. As I tried with my familiar process for new setups, I got the same issue on CloudFront setup of CNAMEs.

The old website was build by a modular WYSIWYG editor, which hosted the site too. It was hosted by an AWS account, I didn't know about. The CloudFront they setup had our CNAME set up, that's why I got the error. After I deleted the old website, it Amplify worked flawlessly.

Thank you for your help! I love Amplify it saves me 2 hours vs. the old setup-workflow.
